### DISCUSSION TITLE
Inline Markdown from another file?

### DISCUSSION THREAD

**Discussion by wizpresso-steve-cy-fan** [slidevjs/slidev#1034] [Q&A]
Sometimes you want to split a large latex equation outside of the main markdown file (for example autogenerated matrix), but the `src` meta seems like it likes to add a new page instead.

**Comment by yogabonito** [comment]
Are there any plans to provide this functionality natively without the need to program a custom preprocessing step? It would not only be useful for equations, but also other big chunks of markdown (e.g., for tables).

So far, I have only seen two similar features:

- including a markdown file representing a [whole slide](https://sli.dev/features/importing-slides)
- including a markdown file as a [snippet](https://sli.dev/features/import-snippet), however, with this approach the markdown is not rendered but shown verbatim instead
